ui/src/views/DashboardView/DashboardView.tsx

```diff
@@ -86,58 +78,55 @@ const DashboardView = () => {
 
   return (
     <List
+      title={
+        <>
+          Zaaktypen&nbsp;
+          {isLoading && <Outline.ArrowPathIcon spin={true} aria-label={'Bezig met laden'} />}
+        </>
+      }
+      formProps={{
+        autoComplete: 'off',
+        fields: [
+          {
+            disabled: loadingCatalogi,
+            defaultValue: 'all',
+            label: 'Selecteer catalogus',
+            options: selectOptions,
+            value: selectedCatalogus,
+            onChange: onCatalogusChange,
+          },
+          {
+            label: 'Filter resultaten',
+            placeholder: 'ZAAKTYPE-XXXX-XXXXXXXXXX',
+            size: 24,
+            value: query,
+            onChange: (e: React.ChangeEvent<HTMLInputElement>) => setQuery(e.target.value),
+          },
+        ],
+        showActions: false,
+      }}
       fields={fields}
       loading={isLoading}
-      results={filteredZaaktypen as unknown as AttributeData<Attribute>[]}
       sort={true}
       onClick={(_: React.MouseEvent, attributeData: AttributeData) => {
         navigate('/zaaktypen/' + uuidExtract(attributeData.url as string));
```

Comment:
This is not going quite right I think: 
the URL is now: `/zaaktypen/e95d9bdf-588d-4965-a469-378d9e0ca91e}/wijzigen`